packages/flutter/lib/src/widgets/widget_span.dart

```diff
@@ -137,7 +138,7 @@ class WidgetSpan extends PlaceholderSpan {
   @override
   int? codeUnitAtVisitor(int index, Accumulator offset) {
     offset.increment(1);
-    return PlaceholderSpan.placeholderCodeUnit;
+    return Unicode.objReplacementChar;
```

Comment:
This and another reference to [PlaceholderSpan.placeholderCodeUnit] are still being changed to [Unicode.objReplacementChar]. Is that intended after https://github.com/flutter/flutter/pull/121298#discussion_r1128260083 ?